examples/tests/sdk_tests/topic/on_message.w

```diff
@@ -0,0 +1,19 @@
+bring cloud;
+
+let t = new cloud.Topic();
+let c = new cloud.Counter();
+
+t.on_message(inflight() => {
+  c.inc();
+});
+
+t.on_message(inflight() => {
+  c.inc();
+});
+
+new cloud.Function(inflight() => {
```

Comment:
@marciocadev I think that a race condition might occur here - take a look at the queue tests implementation and see the trick with the Predicate class:

inflight assert(predicate: Predicate) { 
    let var i = 0;
    while i<600 {
      i = i + 1;
      if predicate.test() {
        assert(predicate.test());
        return;
      } 
      TestHelper.sleep(100);
    }
    assert(predicate.test());
  }